//> error "continue & break statements must be inside of a loop"

import std/core;

func main() -> void {
	continue;
}
